modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/reducers/fieldSetReducer.js

```diff
@@ -16,6 +16,7 @@ import {PagesVisitor} from 'data-engine-js-components-web';
 import {
 	FieldSetUtil,
 	FieldSupport,
+	HandleSectionAdded,
```

Comment:
You are importing a function here, so it should start with with lower case. My suggestion is to use same name found on other imports, that is  `sectionAdded`, just to ease the search for its use.

Reply:
@danvb112  I think this change was undone with the removal of commit containing the new files, would be nice if you amend the last commit with the suggested change